spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -153,6 +153,8 @@ public void undeploy(String id) {
 				}
 			}
 			running.remove(id);
+		} else {
+			throw new IllegalStateException("There is nothing to un-deploy for deployment id " + id);
```

Comment:
This change needs to be removed as we now have aligned the error message for throwing `IllegalStateException` during `undeploy` across deployer implementations.